src/main/java/com/simanavets/tenderflexbackend/entity/Offer.java

```diff
@@ -0,0 +1,25 @@
+package com.simanavets.tenderflexbackend.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+public class Offer {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    @ManyToOne
+    @JoinColumn(name = "user_id")
+    private User user;
+
+    @ManyToOne
+    @JoinColumn(name = "tender_id")
+    private Tender tender;
```

Comment:
We should not use whole Tender object but must use only ID of it just to show relations between them. So all you need is TenderID